ppr-ui/src/components/mhrTransfers/TransferType.vue

```diff
@@ -213,10 +221,34 @@ export default defineComponent({
       localState.declaredValue = val
     })
 
+    watch(() => localState.selectedTransferType, (val:TransferTypeSelectIF) => {
+      switch (val.transferType) {
+        case ApiTransferTypes.SALE_OR_GIFT:
+        case ApiTransferTypes.TO_EXECUTOR_PROBATE_WILL:
+        case ApiTransferTypes.TO_EXECUTOR_UNDER_25K_WILL:
+        case ApiTransferTypes.TO_ADMIN_PROBATE_NO_WILL:
+          localState.transferTypeRules = customRules(
+            required('Select transfer type')
+          )
+          break
+        case ApiTransferTypes.SURVIVING_JOINT_TENANT:
+          localState.transferTypeRules = customRules(
+            required('Select transfer type'),
+            [
+              () => isJointTenancyStructure.value ||
+              'An owner group with joint tenancy is required for this transfer type'
+            ]
+          )
+          break
+      }
+    })
```

Comment:
Similar to the other rule switch cases, we can expand here as we go depending on transfer type.